frontend/packages/dev-console/src/utils/application-utils.ts

```diff
@@ -21,6 +23,10 @@ import {
   DaemonSetModel,
   StatefulSetModel,
 } from '@console/internal/models';
+import {
+  ClusterServiceVersionKind,
+  ClusterServiceVersionModel,
+} from '@console/operator-lifecycle-manager/src';
```

Comment:
we can just have `@console/operator-lifecycle-manager` instead of `@console/operator-lifecycle-manager/src`

Reply:
Doh - sorry... my IDE doesn't resolve the base of the package and wants to do a `/src` when it auto imports. Will fix, thanks!

Reply:
Fixed.